charts/apisix/templates/_pod.tpl

```diff
@@ -115,6 +115,10 @@ spec:
       {{- end }}
       resources:
       {{- toYaml .Values.apisix.resources | nindent 8 }}
+  {{- if .Values.apisix.hostNetwork }}
```

Comment:
Also add this field to readme.

Reply:
done, maybe we can use tools like https://github.com/norwoodj/helm-docs to generate README later?

Reply:
Yes, I agree! Helm-docs is a good tool